### PR TITLE
updated WALinux agent to 2.9.0.4

### PR DESCRIPTION
**Pull request by cjnova**
tested WALinux agent 2.9.0. update

### CODE REVIEW

**Comment by cjnova** [comment]
tested WALinux agent 2.9.0.4 update

File: bicep/main.bicep [hunk 1]
```diff
@@ -72,7 +78,8 @@ param Location string = resourceGroup().location
 
 // Variables
 var TempUsername = 'azureuser'
-var TempPassword = guid(subscription().id,resourceGroup().id)
+// Change password to one knew to be able to login in FreeBSD without root user
```
Comment (dmauser) [inline bicep/main.bicep:81]: This change breaks the deployment. It forces we use the Windows password.

File: bicep/main.bicep [hunk 2]
```diff
@@ -72,7 +78,8 @@ param Location string = resourceGroup().location
...
+var TempPassword = WinPassword
```
Comment (dmauser) [inline bicep/main.bicep:82]: We need to leave the password generated randomly.

**Review by dmauser — CHANGES_REQUESTED** [review]
@cjnova  Can you fix and resubmit based on the comments below? In short, we need to leave var TempPassword to generate random during the installation.

File: bicep/main.bicep (hunk 2 above)
Comment (dmauser) [reply bicep/main.bicep:82]: BTW, thanks for your suggestions to improve this project.

Comment (dmauser) [reply bicep/main.bicep:82]: CCing @Welasco as FYI

**Comment by Welasco** [comment]
Just a heads up for both of you @dmauser  and @cjnova. I'm working on it.

Tks a lot for all the changes @cjnova, I'll test it all.

**Comment by dmauser** [comment]
Closing the pull request per Victor's update.

**Pull request closed without merging**